### ISSUE TITLE
give meaningful names to the folders

### ISSUE THREAD

**Issue by doudou** [rock-core/vscode-rock#44]
Workspace folders can be given a name (I currently edit them by hand when I need to). Give the name relative to the autoproj's workspace root as name by default, maybe adding the workspace name (its basename) if there is more than one autoproj workspace active:

- `tools/roby` if there is only one autoproj workspace.
- `tools/roby (emma)` if there is more than one autoproj workspace.

**Comment by g-arjones** [comment]
Hmm.. I didn't know that, always assumed that was actually editing the real folder's name (a.k.a `mv`). Did you check if there's APIs in vscode for that?

**Comment by doudou** [comment]
> Did you check if there's APIs in vscode for that?

Nope. I really need to get started with other things right now ;-)

Since it is possible by editing the workspace file by hand, I thought that it might be possible through the API, or at least might be something we can extend vscode to do. Worst-case, in the meantime, I thought we could always modify the workspace file.
